src/trio/__main__.py

```diff
@@ -0,0 +1,3 @@
+from trio._repl import main
+
+main(locals())
```

Comment:
```suggestion
variables = locals().copy()
del variables["main"]
main(variables)
```

This is of course only necessary if we change `main` to allow other locals to work. Also probably worth checking that's the only surprising entry!